includes/admin/helper/class-wc-helper-api.php

```diff
@@ -0,0 +1,112 @@
+<?php
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+class WC_Helper_API {
+	public static $api_base;
+
+	/**
+	 * Load
+	 *
+	 * Allow devs to point the API base to a local API development or staging server.
+	 * Note that sslverify will be turned off for the woocommerce.dev + WP_DEBUG combination.
+	 * The URL can be changed on plugins_loaded before priority 10.
+	 */
+	public static function load() {
+		self::$api_base = apply_filters( 'woocommerce_helper_api_base', 'https://woocommerce.com/wp-json/helper/1.0' );
+	}
+
+	/**
+	 * Perform an HTTP request to the Helper API.
+	 *
+	 * @param string $endpoint The endpoint to request.
+	 * @param array $args Additional data for the request. Set authenticated to a truthy value to enable auth.
+	 *
+	 * @return array The respons from wp_remote_request()
+	 */
+	public static function request( $endpoint, $args = array() ) {
+		$url = self::url( $endpoint );
+
+		if ( ! empty( $args['authenticated'] ) ) {
+			self::_authenticate( $url, $args );
+		}
+
+		// Disable SSL verification for the dev environment.
+		if ( defined( 'WP_DEBUG' ) && WP_DEBUG && strpos( self::$api_base, 'https://woocommerce.dev/' ) === 0 ) {
+			$args['sslverify'] = false;
+		}
+
+		// TODO: Check response signatures on certain endpoints.
+		return wp_remote_request( $url, $args );
```

Comment:
`wp_safe_remote_request` should be used here? `woocommerce_helper_api_base` is filterable.